src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp

```diff
@@ -74,7 +74,7 @@ namespace grmhd::AnalyticData {
 class MagnetizedFmDisk
     : public virtual evolution::initial_data::InitialData,
       public MarkAsAnalyticData,
-      private RelativisticEuler::Solutions::FishboneMoncriefDisk {
+      protected RelativisticEuler::Solutions::FishboneMoncriefDisk {
```

Comment:
```
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:278:20: error: 'equation_of_state_type' is a protected member of 'RelativisticEuler::Solutions::FishboneMoncriefDisk'

/home/knelli/Documents/research/spectre/src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp:77:7: note: constrained by protected inheritance here
      protected RelativisticEuler::Solutions::FishboneMoncriefDisk {
      InitialData::equation_of_state_type::thermodynamic_dim;
```

Reply:
Compiler, version? It works for me...

Reply:
Or more of the error? I don't know, I don't have the error so I can't fix it.

Reply:
clang-10 Debug

```
In file included from /home/knelli/Documents/research/spectre/build/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivCleanMagnetizedFmDisk.cpp:6:
In file included from /home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivCleanWithHorizon.hpp:17:
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:278:20: error: 'equation_of_state_type' is a protected member of 'RelativisticEuler::Solutions::FishboneMoncriefDisk'
      InitialData::equation_of_state_type::thermodynamic_dim;
                   ^
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:419:52: note: in instantiation of static data member 'detail::get_thermodynamic_dim<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false>::value' requested here
      detail::get_thermodynamic_dim<initial_data>::value;
                                                   ^
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:424:53: note: in instantiation of static data member 'GhValenciaDivCleanTemplateBase<EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs>, false, false>::thermodynamic_dim' requested here
      hydro::Tags::EquationOfStateFromOptions<true, thermodynamic_dim>,
                                                    ^
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivCleanWithHorizon.hpp:55:35: note: in instantiation of template class 'GhValenciaDivCleanTemplateBase<EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs>, false, false>' requested here
struct EvolutionMetavars : public GhValenciaDivCleanTemplateBase<
                                  ^
/home/knelli/Documents/research/spectre/src/Parallel/Main.hpp:67:35: note: in instantiation of template class 'EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs>' requested here
  using component_list = typename Metavariables::component_list;
                                  ^
/home/knelli/Documents/research/spectre/src/Parallel/CharmMain.tpp:138:9: note: in instantiation of template class 'Parallel::Main<EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs> >' requested here
  (void)charmxx_main_component{
        ^
/home/knelli/Documents/research/spectre/src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp:77:7: note: constrained by protected inheritance here
      protected RelativisticEuler::Solutions::FishboneMoncriefDisk {
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/knelli/Documents/research/spectre/src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/FishboneMoncriefDisk.hpp:162:9: note: member is declared here
  using equation_of_state_type = EquationsOfState::PolytropicFluid<true>;
```

This is the full error

Reply:
Which executable?

Reply:
`EvolveGhValenciaDivCleanMagnetizedFmDisk`